kahuna/public/js/leases/leases.js

```diff
@@ -49,14 +50,26 @@ leases.controller('LeasesCtrl', [
                 ctrl.adding = true;
                 ctrl.newLease.createdAt = new Date();
                 ctrl.newLease.access = ctrl.access;
+                if (!ctrl.showCalendar && ctrl.access === 'deny-syndication') {
+                    ctrl.newLease.startDate = null;
+                    ctrl.newLease.endDate = null;
+                }
+
+                let syndLeases = ctrl.leases.leases.filter((l) =>
```

Comment:
Can `ctrl.leases` ever be null or undefined here? If so this will error and probably needs a guard.

Reply:
It seems to always be set, we never check, see: https://github.com/guardian/grid/blob/a21301fbffad7005f525bb9a456c0459867a18c9/kahuna/public/js/leases/leases.js#L87